manifests/params.pp

```diff
@@ -6,21 +6,23 @@
 #
 class puppetboard::params {
 
-  # puppetlabs/apache names the apache service
-  # resource 'httpd', nevermind the actual os
-  $apache_service = 'httpd'
-
   case $::osfamily {
     'Debian': {
-      if $::operatingsystem == ubuntu {
-        $apache_confd = '/etc/apache2/conf-enabled'
+      if ($::operatingsystem == 'ubuntu') {
```

Comment:
You probably should use upper case naming to be consistent. Unless `$::osfamily` cares about case and `$::operatingsystem` Puppet logic does not

Reply:
can you please switch to the new facts hash? $facts['os']['family']